pkg/registry/debian/debian.go

```diff
@@ -20,6 +20,7 @@ var (
 	registryURL         = "https://deb.debian.org/debian/pool/"
 	buildinfoURL        = "https://buildinfos.debian.net/buildinfo-pool/"
 	binaryReleaseRegexp = regexp.MustCompile(`(\+b[\d\.]+)$`)
+	versionRegex        = regexp.MustCompile(`^(?P<name>[^_]+)_(?P<nonbinary_version>[^_+]+)(?P<binary_version>\+.*)?_(?P<arch>[^_]+)\.deb$`)
```

Comment:
let's also update the "nonbinary_version" name here to be more precise/descriptive